nimutils/sinks.nim

```diff
@@ -308,8 +299,8 @@ proc s3SinkOut(msg: string, cfg: SinkConfig, t: Topic, ignored: StringTable) =
       newPath  = joinPath(state.objPath, newTail)
       response = client.put_object(state.bucket, newPath, msg)
 
-  if response.status[0] != '2':
-    raise newException(ValueError, response.status)
+  if not response.code.is2xx():
+    raise newException(ValueError, response.status & ": " & response.body())
```

Comment:
cc @nettrino adding the response body to the messages as this was rather useful in debugging the issue yesterday. AWS showed (although in XML) that the region header was wrong in the request.... hopefully will aid in debugging errors in the future